### DISCUSSION TITLE
Product recommendation storefrontLabel not showing

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#274] [3. Integrations]
Hi @vovayatsyuk 

We are using Swissup_BreezeMagentoEnterprise with Breeze, Product recommendation title is not showing at the top of products, working fine when exclude from Breeze.

Thanks,
Pankaj

**Comment by vovayatsyuk** [comment]
Try updating Swissup_BreezeMagentoEnterprise. The same issue was fixed 3 hours ago: https://github.com/breezefront/module-breeze-magento-enterprise/commit/80d6f6d6c3bfcbc9db6610322aeac5bc5dbd40eb

**Reply by pk-pankaj** [reply]
Hi @vovayatsyuk 

You fixed the product item name in the given commit but we have an issue with the production recommendation top title which comes from "Display label", please see in attached screen
![image](https://github.com/breezefront/community/assets/19511407/ac601e15-98dc-4b32-9b46-c3d4fea30c72)

**Comment by pk-pankaj** [comment]
@vovayatsyuk 

Worked when I changed the way of binding as you did in the given commit, thanks

```<span class="block-related-heading" role="heading" aria-level="2" data-bind="html : storefrontLabel"></span>```
